src/Place/DummyPlaceProjectionEnricher.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace CultuurNet\UDB3\Place;
+
+use CultuurNet\UDB3\Event\ReadModel\DocumentRepositoryInterface;
+use CultuurNet\UDB3\ReadModel\JsonDocument;
+
+class DummyPlaceProjectionEnricher implements DocumentRepositoryInterface
+{
+    /**
+     * @var DocumentRepositoryInterface
+     */
+    private $repository;
+
+    /**
+     * @var string[]
+     */
+    private $dummyLocationIds = [];
+
+    public function __construct(
+        DocumentRepositoryInterface $repository,
+        array $dummyLocationIds
+    ) {
+        $this->repository = $repository;
+        $this->dummyLocationIds = $dummyLocationIds;
+    }
+
+    public function get($id)
+    {
+        $readModel = $this->repository->get($id);
+        foreach ($this->dummyLocationIds as $dummyLocationId) {
+            $body = $readModel->getBody();
+            if (strpos($body->place->{'@id'}, $dummyLocationId) !== false) {
```

Comment:
This is indeed a URL so good call using strpos instead of an equals comparison here! ✅